src/applications/facility-locator/containers/FacilityLocatorApp.jsx

```diff
@@ -44,9 +44,9 @@ class FacilityLocatorApp extends React.Component {
       <Link to={this.buildSearchString()} key="facility-locator">Find Facilities & Services</Link>,
     ];
 
-    if (location.pathname.match(/facility\/[a-z]+_\d/) && selectedResult) {
+    if (location.pathname.match(/facility\/[a-z]{1,10}_[a-zA-Z0-9]{1,10}/) && selectedResult) {
```

Comment:
please comment this with typical examples that would match this pattern

Reply:
Does this validation get reached if an invalid facility code is entered into the URL?

Reply:
@cehsu I really appreciate this suggestion. I added a number of unit tests to go with a refactored method.